molgenis-core-ui/src/main/frontend/yarn-error.log

```diff
@@ -0,0 +1,6618 @@
+Arguments: 
```

Comment:
i dont think we should add the error log ( maybe added by accident?) , suggest to remove it